modules/assim.batch/R/pda.mcmc.R

```diff
@@ -172,11 +172,11 @@ pda.mcmc <- function(settings, params.id=NULL, param.names=NULL, prior.id=NULL,
     } ## end loop over variables
 
     ## Diagnostic figure
-    if(!is.null(settings$assim.batch$diag.plot.iter) && is.finite(prior.star) && 
+    if(!is.null(settings$assim.batch$diag.plot.iter) && is.finite(prior.star) &&
         (i==start | i==finish | (i %% settings$assim.batch$diag.plot.iter == 0))) {
       pdf(file.path(settings$outdir, paste0('diag.pda', settings$assim.batch$ensemble.id),
         paste0("data.vs.model_", gsub(" ", "0",sprintf("%5.0f", i)), ".pdf")))
-        NEEo <- inputs[[1]]$NEEo
```

Comment:
I changed the` inputs$NEEo` to `inputs$obs` in the latest commit, but forgot to change these in the diagnostic figure plotting chunks